lib/instance.js

```diff
@@ -22,10 +22,9 @@ var initValues = function(values, options) {
     defaults = {};
 
     if (this.Model._hasDefaultValues) {
-      Utils._.each(this.Model._defaultValues, function(valueFn, key) {
-        if (defaults[key] === undefined) {
-          defaults[key] = valueFn();
-        }
+      defaults = _.mapValues(this.Model._defaultValues, function(valueFn) {
+        var value = valueFn();
+        return value._isSequelizeMethod ? value : _.cloneDeep(value);
```

Comment:
Can you investigate further please? I was able to run all unit and integration tests.